src/nxdoc/nuxeo-add-ons/nuxeo-retention-management/nuxeo-retention-installation/nuxeo-retention-installation-standard.md

```diff
@@ -208,22 +209,10 @@ To configure the Retention addon with a dedicated S3 bucket with Object Lock, fi
 
 Records can also be stored in the same bucket as the rest of the content. This provides a simpler architecture, but restricts the possibility to adapt to more complex scenarios in the long run.
 
-[<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More info](#s3-one-bucket-configuration)
-
-{{> anchor 's3-one-bucket-configuration'}}
-#### Storing Records in the Same Bucket as Other Documents
-
-This configuration consists on using the same S3 bucket for records and other documents.
-
-In this case, you can use the Amazon S3 addon with the [default configuration]({{page page='amazon-s3-online-storage'}}) with the Retention addon.
-
-This configuration does NOT allow using the [Amazon S3 Object Lock](https://docs.aws.amazon.com/AmazonS3/latest/dev/object-lock.html) feature (no WORM storage option).
-
 {{#> callout type='warning'}}
-This configuration is only compliant with the binary manager `org.nuxeo.ecm.core.storage.sql.S3BinaryManager`.
```

Comment:
This was wrong, `org.nuxeo.ecm.blob.s3.S3BlobStore` has always been required for retention to work properly